src/datagen/generated/minecolonies/data/minecolonies/colony/quests/general/alchemy.json

```diff
@@ -229,17 +236,18 @@
     {
       "type": "minecolonies:item",
       "details": {
-        "components": {
-          "minecraft:item_name": "{\"translate\":\"com.minecolonies.alchemyquestpotion.name\"}",
-          "minecraft:lore": [
-            "{\"translate\":\"com.minecolonies.alchemyquestpotion.lore\"}"
-          ],
-          "minecraft:potion_contents": {
-            "potion": "minecraft:poison"
-          }
-        },
-        "item": "minecraft:potion",
-        "qty": 1
+        "item": {
+          "components": {
+            "minecraft:item_name": "{\"translate\":\"com.minecolonies.alchemyquestpotion.name\"}",
+            "minecraft:lore": [
+              "{\"translate\":\"com.minecolonies.alchemyquestpotion.lore\"}"
```

Comment:
So those are even in vanilla stringsd instead of proper json?

Reply:
Yeah, that's how vanilla encodes it, for reasons™.

Reply:
probably gonna be another breaking change in "future version". But good enough =D